src/main/java/com/google/devtools/build/lib/exec/SpawnExecException.java

```diff
@@ -71,7 +71,7 @@ public ActionExecutionException toActionExecutionException(
     String message =
         result.getDetailMessage(messagePrefix, getMessage(), isCatastrophic(), forciblyRunRemotely);
     if (verboseFailures) {
-      return new ActionExecutionException(message, this, action, isCatastrophic(), getExitCode());
+      return new ActionExecutionException(message + ": " + getMessage(), this, action, isCatastrophic(), getExitCode());
```

Comment:
From looking at the code I don't quite follow how this fixes it. The value of getMessage() is already appended to message by the implementation of SpawnResult#getDetailMessage. Why does appending it twice help?

Reply:
No, I think in most case, getDetailMessage doesn't actually append the value of getMessage().
See 
https://github.com/bazelbuild/bazel/blob/2d0046a39e383613dac12b2dfe9845dd22df100f/src/main/java/com/google/devtools/build/lib/actions/SpawnResult.java#L343

Reply:
Ah looked too quick :). Did you consider fixing this in getDetailMessage instead of here? I think this would allow us to avoid printing the same error message twice?

Reply:
Make sense!

Reply:
Please take a look again. ;)